gossip/tips_cache.h

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright (c) 2018 IOTA Stiftung
+ * https://github.com/iotaledger/entangled
+ *
+ * Refer to the LICENSE file for licensing information
+ */
+
+#ifndef __GOSSIP_TIPS_H__
+#define __GOSSIP_TIPS_H__
+
+#include "common/errors.h"
+#include "common/trinary/flex_trit.h"
+#include "utils/containers/hash/hash243_set.h"
+#include "utils/handles/rw_lock.h"
+
+#define MAX_TIPS 5000
```

Comment:
not used, remove

Reply:
It's being used on #491

Reply:
Fair enough; as this is just stored in the `capacity` member I'd much rather not have this in a global header but in the source file that initialises the tips_cache?

Reply:
Sure. As it can affect the result of `getTips` API call, should it be configurable with an option from CLI ?

Reply:
:thinking: there's two distinct params here - how many tips are returned from API call and how many are stored in memory. I believe the two are tracked independently on IRI, not sure

(but ya, might be good)

Reply:
Just looking at IRI conf code, I don't see any option to limit the number of tips returned by the call

Reply:
CLI option will be added in #491